pkg/jobs/jobs_test.go

```diff
@@ -2906,3 +3093,23 @@ func TestLoseLeaseDuringExecution(t *testing.T) {
 	registry.TestingNudgeAdoptionQueue()
 	require.Regexp(t, `expected session '\w+' but found NULL`, <-resumed)
 }
+
+func checkBundle(t *testing.T, zipFile string, expectedFiles []string) {
+	t.Helper()
+	r, err := zip.OpenReader(zipFile)
+	require.NoError(t, err)
+
+	// Make sure the bundle contains the expected list of files.
+	filesInZip := make([]string, 0)
+	for _, f := range r.File {
+		if f.UncompressedSize64 == 0 {
+			t.Fatalf("file %s is empty", f.Name)
+		}
+		filesInZip = append(filesInZip, f.Name)
+	}
+	sort.Strings(filesInZip)
+	sort.Strings(expectedFiles)
+	if fmt.Sprint(filesInZip) != fmt.Sprint(expectedFiles) {
```

Comment:
nit: I think there's a `require.Equal` that could be helpful here.

Reply:
done.